Jenkinsfile

```
@@ -40,5 +40,5 @@
 // I.e. for testing library changes
 //@Library(value="pipeline-lib@your_branch") _
 
-packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'], 'test-tag': 'mfu'])
-
+packageBuildingPipelineDAOSTest(['distros' : ['el8', 'leap15'],
+                                 'test-tag': 'mpifileutils copy_procs dcp dsync dserialize'])
```

Now that 2.0 is EOL, this is better
```suggestion
                                 'test-tag': 'mfu'])
```

---

Or, if we want just VM and HW Small: `mfu,-hw mfu,hw,small`

---

You'll only get HW Small in any case as the pipeline only runs on that HW platform.

---

@daltonbohning Thank you! I reverted my change in Jenkinsfile.